app/src/main/res/values/dimens.xml

```diff
@@ -41,6 +41,7 @@
     <dimen name="dp_200">200dp</dimen>
     <dimen name="sp_8">8sp</dimen>
     <dimen name="dp_25">25dp</dimen>
+    <dimen name="dp_180">180dp</dimen>
```

Comment:
WARNING: [UnusedResources] The resource `R.dimen.dp_180` appears to be unused
 Unused resources make applications larger and slow down builds.